imagersite/imager_profile/templates/imager_profile/profile.html

```diff
@@ -0,0 +1,40 @@
+{% extends "imagersite/base.html" %}
+{% load static %}
+
+
+{% block nav-bar %}
+<a href="{% url 'library' %}" class="btn btn-primary btn-lg">Library</a>
+<a href="{% url 'home' %}" class="btn btn-primary btn-lg">Home</a>
+{% endblock %}
+
+
+{% block content %}
+{% if user == profile.user %}
```

Comment:
You shouldn't have to hide all of this from a different user, I should be able to see basic details like their name, photography level etc.